src/scrape_up/twitter/profile.py

```diff
@@ -0,0 +1,132 @@
+from time import sleep
+from selenium.webdriver.common.keys import Keys
+from selenium.webdriver import Chrome
+from webdriver_manager.chrome import ChromeDriverManager
```

Comment:
Can't accept more PR with `selenium.webdriver` because it is not compatible with all the devices as well as on cloud platforms.

We already have plenty of modules with the same issue, so look for any alternative way to this. Else close this PR.